source/FloydPink.Flickr.Downloadr.UI/Windows/PreferencesWindow.cs

```diff
@@ -108,8 +108,9 @@ private void setFieldsFromModel(Preferences preferences)
     {
       Log.Debug("setFieldsFromModel");
       // Filename
-      radioPhotoId.Active = !preferences.TitleAsFilename;
```

Comment:
IMO, this is a little confusing - maybe both these boolean variables can be replaced with a new enumeration with three values (because there are three modes for the file name now)?